shared/utils/replaceLinks.ts

```diff
@@ -12,12 +12,16 @@ const path = `(?:\\/(?:[+~%/_.${unicode}-]*[+~%/${unicode}-]+)?)?`
 const query = `(?:\\?([-+=&;%@.${unicode}]*[-+=&;%@${unicode}]+)?)?`
 // language=RegExp
 const hash = `(?:#([.!/\\\\${unicode}]*[!/\\\\${unicode}]+)?)?`
-const mail = /[.\-;:&=+$,\w]+@([A-Za-z0-9-]+(?:\.[A-Za-z0-9-]+)*)/
+const mail = /[.\-;:&=+$,\w]{1,64}@([A-Za-z0-9-]+(?:\.[A-Za-z0-9-]+)*)/
```

Comment:
❓ Why did you change this? Is this the maximum allowed length of email addresses?

Reply:
Here:
https://github.com/digitalfabrik/integreat-app/pull/3339/files/5d9c74da8902b148f1b1c6a8cfe950971d36e5fc#r2157001255